lib/cartopy/feature.py

```diff
@@ -174,7 +175,17 @@ def __init__(self, category, name, scale, **kwargs):
         self.name = name
         self.scale = scale
 
+        # Use autoscaling if scale == 'a', 'auto' or 'autoscale'
+        if scale.lower() in ('a', 'auto', 'autoscale'):
```

Comment:
Please just accept ``auto``. Not case sensitive, no ``a`` nor ``autoscale``. Let's not encourage people to write the wrong thing.